src/Validators/SKUValidator.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Portal\Validators;
+
+class SKUValidator
+{
+    private const SKU_REGEX = '/^[a-z0-9A-Z]{10,20}$/';
```

Comment:
you can make it case-insensitive instead of including both a-z and A-Z, but that's just personal preference